packages/hash/frontend/src/testUtils/testUtils.tsx

```diff
@@ -1,20 +1,29 @@
 import { MockedProvider, MockedResponse } from "@apollo/client/testing";
+import { createTheme, ThemeProvider } from "@mui/material";
 // eslint-disable-next-line no-restricted-imports
 import { render, RenderOptions } from "@testing-library/react";
 
+import { borderRadii } from "../theme/borderRadii";
+import { palette } from "../theme/palette";
+import { boxShadows, dropShadows } from "../theme/shadows";
+
 type CustomRenderOptions = RenderOptions & {
   mocks?: MockedResponse[];
 };
 
+const theme = createTheme({ palette, boxShadows, dropShadows, borderRadii });
+
 const customRender = (
   ui: React.ReactElement,
   options: CustomRenderOptions = {},
 ) => {
   const { mocks = [] } = options;
   const Wrapper: React.FC = ({ children }) => (
-    <MockedProvider mocks={mocks} addTypename={false}>
-      {children}
-    </MockedProvider>
+    <ThemeProvider theme={theme}>
```

Comment:
A `ThemeProvider` reusing the existing MUI theme is added to fix the `jest` tests.

Reply:
Why not import the `theme` that's already created in `src/theme/index.ts`? It saves having to update it in two places, and guarantees we're testing the theme we're using.

Reply:
This was addressed in https://github.com/hashintel/hash/pull/378/commits/5fd1f743cdb251628c80e685824f16db3bd52407